code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -66,8 +66,8 @@
 		if(MODE_RAD)
 			show_rads()
 
-/obj/item/clothing/glasses/meson/engine/proc/show_rads()
-	var/mob/living/carbon/human/user = loc
+/mob/proc/show_rads(range)
+	var/mob/user = src
```

Comment:
This line should be removed and the instances of `user` should be replaced with `src`, except line 92, where `user.client` would just become `client` because implicit src.

Reply:
Fixed in c9e796d